dissect/target/helpers/regutil.py

```diff
@@ -707,7 +724,7 @@ def value(self) -> ValueType:
         return self.kv.value
 
     @property
-    def type(self) -> RegistryValueType:
+    def type(self) -> RegistryValueType | int:
```

Comment:
```suggestion
    def type(self) -> RegistryValueType:
```
No longer necessary.